stories/css-shadow-variable-clash.tsx

```diff
@@ -0,0 +1,19 @@
+import { styled } from '@compiled/react';
+
+export default {
+  title: 'css prop/variable clash',
+};
+
+const isPrimary = false;
+const Component = styled.div`
+  color: ${({ isPrimary }) => (isPrimary ? 'green' : 'red')};
+`;
+console.log(isPrimary);
```

Comment:
leftover debug code

Reply:
I think that was the quickest workaround to use the variable, otherwise the build will fail.

Reply:
I have removed the console.log and all checks has passed.